public/app/plugins/datasource/cloudwatch/query-runner/CloudWatchLogsQueryRunner.ts

```diff
@@ -342,7 +342,12 @@ export class CloudWatchLogsQueryRunner extends CloudWatchRequest {
     };
 
     return queryFn(requestParams).pipe(
-      map((response) => response.data),
+      map((response) => {
+        if (response.errors) {
+          throw new Error(response.errors[0].message);
```

Comment:
This thrown error is showing up in the console now.

I think it would be better if we could just return `response` here and let the built in error handling show the error message. I think we also may be missing out on being able to associate errors with their respective ref ids by only taking the first error message.

We may need to change the signature of `makeLogActionRequest` to return `Observable<DataQueryResponse>`, but not sure how much work that is to modify all the places that this function is called from.